dotcom-rendering/src/components/Card/Card.tsx

```diff
@@ -326,7 +327,9 @@ export const Card = ({
 				commentCount={
 					discussionId !== undefined ? (
 						<Link
-							data-discussion-id={discussionId}
+							{...{
+								[DISCUSSION_ID_DATA_ATTRIBUTE]: discussionId,
+							}}
```

Comment:
We've created a wide coupling here.

* `Card` depends on `useCommentCount.DISCUSSION_ID_DATA_ATTRIBUTE`
* `Card` depends on `CardCommentCount.importable`
* `CardCommentCount.importable` depends on `useCommentCount`

Assuming this is mainly because we want the Island to be as small as possible. Not sure what we can do, just thought it worth having a think about

Reply:
As long as the dependency is not cyclical, I think that’s fine?

```mermaid
flowchart TD
    Card --> CardCommentCount.importable & useCommentCount
    CardCommentCount.importable --> useCommentCount
    useCommentCount
```